docker-compose-azure.yml

```diff
@@ -0,0 +1,18 @@
+version: '0.1'
+
+services:
+ efm-mocks:
```

Comment:
her er berre docker-bygg for efm-mocks lagt inn - meiner det var fleire docker-bygg som skulle med ?

eller er tanken kanskje at du ikkje treng bygge docker-bygg i det heile iom. at du gjer det i skriptet via docker-compose up ? i så fall går det an å slå av docker-bygg og fjerne docker-compose-azure.yml

funker det å bygge rotmappa med "build: ." ? ingen dockerfile der? bygger den rekursivt då? eller plukker opp docker-compose.yml ?

Reply:
docker-compose-azure.yml kan fjernes sidan du ikkje bygger docker-artifact - skal ingen av desse mock-komponentane f.eks. i testmiljø ?

Reply:
Tanken er at scriptet tar seg av docker-compose up som startar alle modulane sine docker bygg . Mock komponentane skal ikkje til testmiljøet. 

Då kan eg jo fjerne docker-compose-azure.yml